test/Microsoft.ML.Tests/TrainerEstimators/MatrixFactorizationTests.cs

```diff
@@ -146,21 +146,21 @@ internal class MatrixElement
         {
             // Matrix column index starts from 1 and is at most _synthesizedMatrixColumnCount.
             // Contieuous=true means that all values from 1 to _synthesizedMatrixColumnCount are allowed keys.
-            [KeyType(Contiguous = true, Count = _synthesizedMatrixColumnCount, Min = _synthesizedMatrixFirstColumnIndex)]
+            [KeyType(Count = _synthesizedMatrixColumnCount)]
             public uint MatrixColumnIndex;
             // Matrix row index starts from 1 and is at most _synthesizedMatrixRowCount.
             // Contieuous=true means that all values from 1 to _synthesizedMatrixRowCount are allowed keys.
-            [KeyType(Contiguous = true, Count = _synthesizedMatrixRowCount, Min = _synthesizedMatrixFirstRowIndex)]
+            [KeyType(Count = _synthesizedMatrixRowCount)]
             public uint MatrixRowIndex;
             // The value at the MatrixColumnIndex-th column and the MatrixRowIndex-th row in the considered matrix.
             public float Value;
         }
 
         internal class MatrixElementForScore
         {
-            [KeyType(Contiguous = true, Count = _synthesizedMatrixColumnCount, Min = _synthesizedMatrixFirstColumnIndex)]
+            [KeyType(Count = _synthesizedMatrixColumnCount)]
```

Comment:
>_synthesizedMatrixColumnCount [](start = 29, length = 29)

This should be `_synthesizedMatrixColumnCount +_synthesizedMatrixFirstColumnIndex` because `Count` stands for maximum key index allowed now. Do you think we could rename `Count` to `Max`?

Reply:
I'd rather we talk about the renaming as a separate issue, since that would be a very disruptive change. That is, we'd have this one (comparatively) simple change to remove the old capability, then the renaming as a separate PR. Even if we thought this was a good idea (and we very well may) I would still prefer to have that in two commits against master.